packages/cli/src/Server.ts

```diff
@@ -2907,6 +2907,10 @@ export async function start(): Promise<void> {
 			},
 			deploymentType: config.get('deployment.type'),
 			binaryDataMode: binarDataConfig.mode,
+			n8n_multi_user_allowed:
+				config.get('userManagement.disabled') === false ||
+				config.get('userManagement.hasOwner') === true,
+			smtp_set_up: config.get('userManagement.emails.mode') === 'smtp',
```

Comment:
Only telemetry payloads seem to require snake_case. Are `n8n_multi_user_allowed` and `smtp_set_up` required to be in snake_case here and in the interface?

Reply:
This is an object that is only used for telemetry; if you check the other keys they are also snake case. This is instantiated here and used once to identify the instance. Thanks for bringing this up!